docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+### Submitting new documentation
+
+Submitting new documentation generally involves submitting new [Tutorials](/3.0.0-beta.x/community/contribute-with-docs.html#tutorials) or new [How-tos](/3.0.0-beta.x/community/contribute-with-docs.html#how-to-s).
+
+**NOTE:** If you are providing _new examples_ for existing [Reference Topics](http://localhost:8080/documentation/3.0.0-beta.x/community/contribute-with-docs.html#reference-topics), then the next section on [adding, modifying and deleting content](http://localhost:8080/documentation/3.0.0-beta.x/community/contribute-with-docs.html#adding-modifying-deleting-content-from-the-docs) applies.
+
+- Before you start to determine if you're writing a new **Tutorial** or a new **How-to**
+- Outline the **Tutorial** or a new **How-to**.
+- Speak to a Strapi Maintainer or open an issue on GitHub with the outline to receive feedback.
+- Create a GitHub repo with your app for the **Tutorial** or for the code for the **How-to**.
+- Show this code to the Strapi Maintainer and address any feedback
+- Following the [Docs Style Guide](http://localhost:8080/documentation/3.0.0-beta.x/community/contribute-with-docs.html#docs-style-guide) below, write your Tutorial and make a pull request.
+
+#### Tips
+
+1. Make sure that following the steps results in a working app (for Tutorials) or working code (for How-tos).
+2. Smaller workable apps are better than large apps that have many moving parts.
+3. If you have any questions, please ask!
+
+### Adding / Modifying / Deleting Content from the Docs
+
+_Adding, modifying, and deleting content_ from the documentation follows the [guidelines for making a Pull Request](/3.0.0-beta.x/community/contribute-with-code.html#before-submitting-a-pull-request) to the Strapi monorepo. For correcting typos, or clarifications, this is a relatively straightforward process.
+
+**Tutorials** and **How-tos** are sometimes by a members of the community. A byline appears towards the title with a **Written by: Paul Bocuse**. If you make _significant_ revisions to the **Tutorial** or **How-tos**, then the byline would change and include your name, like so: **Original Author: Paul Bocuse, Revised by: Your Name**. You should link to your GitHub from your name.
+
+**Examples** are generally added to **Reference Topics** and **How-tos**. The examples are added directly in the existing content.
+
+### Submitting an Externally Hosted Tutorial
+
+If you have written a **Tutorial**, **How-to** or other teaching material that is _outside_ of the documentation. Then you may submit a pull request with a link to your guide on the [Articles](/3.0.0-beta.x/articles/) page.
+
+### Docs Style Guide
+
+This style guide exists to show and explain the standards and style of how our documentation is written. We are offering this style guide to help you stay consistent with the existing documentation.
```

Comment:
This style guide exists to show and explain the standards and style of how our documentation is written. It has been redacted so each contribution stays consistent with the existing documentation.

Reply:
That second sentence could be better. But `redacted` doesn't quite mean like the french word. in English it means to go through a document and remove words, like confidential names, etc.

I clarified it like this:

```
This style guide exists to show and explain the standards and style of how our documentation is written. Its purpose is to help you stay consistent with the existing documentation.

```